### ISSUE TITLE
Update examples for Electron Forge

### ISSUE THREAD

**Issue by goosewobbler** [webdriverio-community/wdio-electron-service#58]
We should use Electron Forge for building the main two example apps (`example` & `example-cjs`).

Proposing a new `example-electron-builder` (ESM - duplicate of `example`) for testing an app built with EB.

https://www.electronforge.io/
https://www.electron.build/

**Comment by goosewobbler** [comment]
Having an explicit electron-builder example was considered but would require another duplicate directory (e.g. `example-electron-builder`) and would provide limited benefit.

**Comment by goosewobbler** [comment]
This is halfway to complete, blocked on Forge ESM support - see #243

**Comment by goosewobbler** [comment]
Implemented in 6.0.1
